upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -12,6 +12,7 @@ Use one of the following methods to upgrade your HCP clusters:
 
 * Upgrade only your hosted control plane. This does not impact your worker nodes.
 * Upgrade only your machine pool. This initiates a rolling reboot of a specific machine pool and temporarily impacts the worker nodes on the specific machine pool. It does not impact all your worker nodes if you have multiple machine pools.
+* Upgrade multiple machine pools simultaneously. This will initiate a rolling reboot of just updated machine pools. This allows for updating multiple nodes simultaneously within a cluster.
```

Comment:
> just updated machine pools

I'm not sure of what this means. 
Only updated machine pools?
Recently updated machine pools?

Reply:
Hello @mburke5678. Good question. I asked PM and he agreed it needed to be clarified. Will make changes based on the info he provided.  Thx.